Fortinet/fortiproxy/_meta/smart-descriptions.json

```
@@ -769,5 +769,25 @@
         "value": "dlp"
       }
     ]
+  },
+  {
+    "value": "Intrusion {action.name} From {source.ip} with message {fortinet.msg}",
+    "relationships": [
+      {
+        "source": "source.ip",
+        "target": "url.full",
+        "type": "connected to"
+      }
+    ],
+    "conditions": [
+      {
+        "field": "event.kind",
+        "value": "utm"
+      },
+      {
+        "field": "event.category",
+        "value": "ips"
```

The condition doesn't match the smart-description
```suggestion
      {
        "field": "source.ip"
      },
      {
        "field": "fortinet.msg"
      },
      {
        "field": "action.name"
```